.ci/Jenkinsfile

```
@@ -78,7 +78,7 @@ pipeline {
           }
         }
         stage('Integration Tests') {
-          agent { label 'linux && immutable' }
+          // agent { label 'linux && immutable' } Reusing top level build agent
```

parallel stages need an agent to be executed in parallel

```suggestion
          agent { label 'linux && immutable' } Reusing top level build agent
```

---

I'm moving unit tests outside of the parallel, running there just the integration ones